Dockerfile

```diff
@@ -3,10 +3,10 @@ FROM madnificent/ember:3.20.0 as builder
 LABEL maintainer="info@redpencil.io"
 
 WORKDIR /app
-COPY package.json .
-RUN npm install
+COPY package.json package-lock.json ./
+RUN npm ci
 COPY . .
-RUN ember build -prod
+RUN ember build
```

Comment:
we should try to get this fixed still, I believe this was due to an issue with the datastore not working correctly in the editor?